client/tests/acceptance/user-can-click-landuse-form-edit-test.js

```diff
@@ -0,0 +1,132 @@
+import { module, test } from 'qunit';
+import {
+  visit,
+  click,
+  currentURL,
+  settled,
+  fillIn,
+} from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { authenticateSession } from 'ember-simple-auth/test-support';
+
+const saveForm = async () => {
+  await click('[data-test-save-button]');
+  await settled();
+};
+
+module('Acceptance | user can click landuse form edit', function(hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(() => {
+    authenticateSession({
+      emailaddress1: 'me@me.com',
+    });
+  });
+
+  test('User can add an applicant on the landuse form', async function(assert) {
+    this.server.create('project', 1, {
+      packages: [this.server.create('package', 'toDo', 'landuseForm')],
+    });
+
+    await visit('/landuse-form/1/edit');
+
+    await click('[data-test-add-applicant-button]');
+    await fillIn('[data-test-input="dcpFirstname"]', 'Tess');
+    await fillIn('[data-test-input="dcpLastname"]', 'Ter');
+    await fillIn('[data-test-input="dcpEmail"]', 'tesster@planning.nyc.gov');
+    await click('[data-test-save-button]');
+
+    assert.equal(this.server.db.applicants.firstObject.dcpFirstname, 'Tess');
```

Comment:
I guess we were asserting against the fake db to check if the data had actually saved -- but it might be better to transition to the `landuse-form.show` route and then assert against the dom there. I don't think the `landuse-form.show` route is built out yet, but maybe part of building that out will be updating these tests to check for the presence of the correct data in the dom.

Reply:
Sounds good! Yeah, we can trust that Ember Data will issue the correct request when saving.